src/services/class.service.js

```diff
@@ -92,8 +129,11 @@ const findSimilarClasses = async (currentClass, teacher, query) => {
     level: { $gte: Math.min(1, currentClass.level - 1), $lte: Math.max(currentClass.level + 1, 10) }
```

Comment:
this needs to be equal can you fix that as well

Reply:
the level should be exactly equal?